bluemira/magnetostatics/polyhedral_prism.py

```diff
@@ -336,8 +361,8 @@ def field_fabbri(
     """
     field = np.zeros(3)
     for i, normal in enumerate(face_normals):
```

Comment:
if I do this I get rid of that numba performance warning. I dunno if there is a better way...
```suggestion
    for i, _normal in enumerate(face_normals):
        normal = np.ascontiguousarray(_normal)
```